app/src/main/java/org/p2p/wallet/feerelayer/model/TokenInfo.kt

```diff
@@ -1,6 +1,12 @@
 package org.p2p.wallet.feerelayer.model
 
+import org.p2p.wallet.utils.Constants.WRAPPED_SOL_MINT
+
 class TokenInfo(
     val address: String,
     val mint: String
-)
+) {
+
+    val isSOL: Boolean
+        get() = WRAPPED_SOL_MINT == mint
```

Comment:
isSol can be a field. class and its fields are final, so it's okay to initialise value on creating instance

Reply:
That's true, will do